### PR TITLE
feat: Enable group admin to make other members admin

### LINKED ISSUE #438: Enable group admin to make other members admin
As a group admin i want to be able to promote other group members to become admin.

### PR DESCRIPTION
**Pull request by TripleSpeeder**
fixes #438

### CODE REVIEW

File: BrightID/src/components/GroupsScreens/Members/MemberCard.js [hunk 1]
```diff
@@ -1,52 +1,109 @@
...
+
+  // show flagged status of member?
+  useEffect(() => {
+    if (!userIsAdmin) {
+      // only admins can see flags
+      setFlagged(false);
+    } else {
+      setFlagged(flaggers && Object.keys(flaggers).length);
+    }
+  }, [flaggers, userIsAdmin]);
```
Comment (adamstallard) [inline BrightID/src/components/GroupsScreens/Members/MemberCard.js:86]: Why can only admins see flags?

File: BrightID/src/components/GroupsScreens/Members/MembersScreen.js [hunk 2]
```diff
@@ -65,6 +66,36 @@ export class MembersScreen extends Component<Props, State> {
...
+      {
+        text: 'OK',
+        onPress: async () => {
+          const { route, dispatch } = this.props;
+          const group = route.params?.group;
+          try {
+            await api.addAdmin(user.id, group.id);
+            await dispatch(addAdmin(user.id, group));
+          } catch (err) {
+            Alert.alert('Error dismissing member from the group', err.message);
```
Comment (adamstallard) [inline BrightID/src/components/GroupsScreens/Members/MembersScreen.js:84]: This alert message doesn't seem to match the context of the error

File: BrightID/src/components/GroupsScreens/Members/MemberCard.js (hunk 1 above)
Comment (TripleSpeeder) [reply BrightID/src/components/GroupsScreens/Members/MemberCard.js:86]: > Why can only admins see flags?

I had the exact same question. Mahdi answered in keybase: keybase://chat/brightid#frontend/9286

File: BrightID/src/components/GroupsScreens/Members/MembersScreen.js (hunk 2 above)
Comment (TripleSpeeder) [reply BrightID/src/components/GroupsScreens/Members/MembersScreen.js:84]: Nice find!

**Comment by TripleSpeeder** [comment]
Rebased...

**Pull request merged**